src/testing/flow_solver_cases/1D_burgers_rewienski_snapshot.cpp

```diff
@@ -118,7 +118,7 @@ void BurgersRewienskiSnapshot<dim, nstate>::steady_state_postprocessing(std::sha
                 sensitivity_dWdb[i]);
     }
     solutions_table.set_precision("Sensitivity:", 16);
-    std::ofstream out_file(this->all_param.flow_solver_param.sensitivity_table_filename + ".txt");
+    std::ofstream out_file(this->unsteady_data_table_filename_with_extension);
```

Comment:
This should be left to the value it was before, sensitivity_table_filename.

Reply:
My mistake, sorry about that

Reply:
I meant to make this change in `compute_unsteady_data_and_write_to_table` within this file; I've address this